src/string_object.cpp

```diff
@@ -516,11 +537,19 @@ nat_int_t StringObject::index_int(Env *env, Value needle, size_t start) const {
     return ptr - c_str();
 }
 
-Value StringObject::initialize(Env *env, Value arg) {
+// TODO: according to documentation, "the given capacity is
+//  advisory only, and may or may not set the size of the internal
+//  buffer, which may in turn affect performance".  This said, it
+//  should be possible to set the m_capacity field of the
+//  underlying String object in the future.
```

Comment:
I don't think the we need the TODO note here. The `capacity` is an implementation detail and I'm fine with us ignoring it indefinitely. 😄